### PR TITLE
Continue migrating #135 (letter 'n')

### PR DESCRIPTION
**Pull request by luzpaz**
It seems I forgot to open a PR for this one.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5342,26 +5374,46 @@ notasion->notation
...
+notifiying->notifying
 nott->not
 notwhithstanding->notwithstanding
+not either->neither
+not quit->not quite
 noveau->nouveau
 Novermber->November
 nowdays->nowadays
 nowe->now
+no compliant->noncompliant
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: this can be used legitimately

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: True. These are good examples of #255 
I wonder if there is a way we can consider entering them in to codespell somehow and use some regex look aheads to utilize them?

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: Not in this PR

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: fair enough

**Comment by luzpaz** [comment]
removed the offending entry. Ready to be merged.

**Pull request merged**